tests/Console/E2ETest.php

```diff
@@ -184,8 +188,23 @@ private function installComposerDeps(): void
             // Install deps only if there's none
             $this->assertNotEmpty(getenv('PATH') ?: getenv('Path'), 'E2E tests need a system composer installed, but it could not be found without a PATH set');
 
-            $process = new Process(sprintf('%s %s', (new ComposerExecutableFinder())->find(), 'install'));
-            $process->mustRun();
+            try {
+                $process = new Process(sprintf('%s %s', (new ComposerExecutableFinder())->find(), 'install'));
```

Comment:
Passing a command as string is deprecated since symfony 4.2
https://github.com/symfony/process/blob/master/Process.php#L146

could you please change this behaviour on our side?

Reply:
I'll make another PR with all such instances fixed.